tck/pom.xml

```diff
@@ -49,36 +49,16 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.eclipse.microprofile.apis</groupId>
-            <artifactId>microprofile-config_1.0_api</artifactId>
-            <version>1.0-SNAPSHOT</version>
-            <scope>provided</scope>
+            <groupId>org.eclipse.microprofile.config</groupId>
+            <artifactId>microprofile-config-api</artifactId>
+            <version>${project.version}</version>
         </dependency>
-        
         <dependency>
-            <groupId>javax.enterprise</groupId>
-            <artifactId>cdi-api</artifactId>
-            <version>1.2</version>
+            <groupId>javax</groupId>
```

Comment:
Can we avoid depending on the entirety of javaee?  I see it's done in a few places.